bin.src/make_latiss_export.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# This file is part of prompt_prototype.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+"""Selectively export the contents of the LATISS dataset.
+
+This script selects some LATISS data in a source butler repo, and makes an export
+file for importing to the test central prompt processing repository.
+"""
+
+
+import argparse
+import logging
+import sys
+
+import lsst.daf.butler as daf_butler
+from lsst.utils.timer import time_this
+
+
+def _make_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--src-repo",
+        required=True,
+        help="The location of the repository from which datasets are exported.",
+    )
+    return parser
+
+
+def main():
+    logging.basicConfig(level=logging.INFO, stream=sys.stdout)
+
+    args = _make_parser().parse_args()
+    src_butler = daf_butler.Butler(args.src_repo)
+
+    with time_this(msg="Exporting datasets and collections", level=logging.INFO):
```

Comment:
Note that this message is printed _after_ the export is finished.

Reply:
Message corrected